CHANGELOG.md

```diff
@@ -3,7 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+## [0.15.0] - 2019-12-04
+
+### Fixed
+
+- Don't delete / disable Users if the Change the DN - [#470](https://github.com/owncloud/user_ldap/issues/470)
```

Comment:
`if the Change the DN` - this does not make sense to me, I have to try and guess what it might mean.

Reply:
Use Case: A LDAP user is moved in AD Server because of changing a department